services/organizer_service.js

```diff
@@ -0,0 +1,43 @@
+const { models } = require('../db/relations')
+const { Checker } = require('./data_checker')
+const { Authorize } = require('./jwt_service')
+const Organizer = models.Organizer
+const { Responses } = require('../utils/responses')
+const { Meeting } = require('../db/meeting')
+const createResponse = Responses.createResponse
+
+const invalidResponse = createResponse("Invalid login or password", 400)
+
+const findOrganizerWithEmail = async (email) => await Organizer.findAll({ where: { email: email } })
+
+const organizerService = {
+    register: async (email, password) => {
+        const checkResult = Checker.checkEmail(email) 
+        if(checkResult!==true)return createResponse(checkResult,400)
+        const organizers = await findOrganizerWithEmail(email)
+        if(organizers.length!==0) return createResponse("This mail is already used",400) 
+        const newPassword = await Authorize.hashPassword(password)
+        await Organizer.create({ email: email, password: newPassword })
+        return createResponse("Created user",201)
+    },
```

Comment:
Imo spoko jest po jakichś returnach zrobić linijkę przerwy, żeby zaznaczyć, że jest kilka miejsc w których można wyjść z funkcji, bo ciężko się to czyta jak się nie wie jak to bangla, ale to tylko ja tak myślę. Pewnie się tak nie robi. No nie wiem xd.

Jeszcze trochę Ci się rozjeżdża koncepcja spacjowania, tutaj if(i==j), niżej if (i==j), wyżej if (i == j). Do ujednolicenia, można jakiegoś prettiera skonfigurować, żeby się pluł o takie rzeczy

Reply:
Potrzebna opinia biegłych @tomros766 @Davenury